### PR TITLE
add summary for release notes

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

- add summary for release notes and tiup docs
- convert CRLF to LF
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ecosystem-tool-user-case.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-当需要将 TiDB 集群的数据迁出到其他 TiDB 集群时，可使用 [Dumpling](/dumpling-overview.md) 从 TiDB 将全量数据导出为 SQL dump files，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB。
-
-如果还需要执行增量数据的迁移，则可使用 [TiCDC](/ticdc/ticdc-overview.md)。
-
-## TiDB 增量数据订阅
-
-当需要订阅 TiDB 增量数据的变更时，可使用 [TiCDC](/ticdc/ticdc-overview.md)。
+---
+title: TiDB 工具的使用场景
+summary: 本文档介绍 TiDB 工具的常见使用场景与工具选择。
```
Comment (lilin90) [inline ecosystem-tool-user-case.md:3]: 看起来有几篇文档（如这篇）并没改动什么，但显示效果像是改了很多似的🤔

Comment (Oreoxmt) [reply ecosystem-tool-user-case.md:3]: @lilin90 可以勾选 Hide whitespace
image

File: releases/release-3.0.11.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB 3.0.11 Release Notes
 aliases: ['/docs-cn/dev/releases/release-3.0.11/','/docs-cn/dev/releases/3.0.11/']
+summary: TiDB 3.0.11 发布，包含兼容性变化、新功能和 Bug 修复。新增配置项 `max-index-length` 控制索引最大长度，显示分区表的分区元信息，以及 TiDB 集群之间数据双向复制功能。Bug 修复包括查询结果不正确、Goroutine 泄露等问题。TiKV 也进行了日志输出优化和问题修复。 TiDB Ansible 修复了失效文档链接和未定义变量问题。
```
Comment (lilin90) [inline releases/release-3.0.11.md:4]:
```suggestion
summary: TiDB 3.0.11 发布，包含兼容性变化、新功能和 Bug 修复。新增配置项 `max-index-length` 控制索引最大长度，显示分区表的分区元信息，以及 TiDB 集群之间数据双向复制功能。Bug 修复包括查询结果不正确、Goroutine 泄露等问题。TiKV 也进行了日志输出优化和问题修复。TiDB Ansible 修复了失效文档链接和未定义变量问题。
```

File: releases/release-4.0.2.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB 4.0.2 Release Notes
 aliases: ['/docs-cn/dev/releases/release-4.0.2/']
+summary: TiDB 4.0.2 发布，兼容性改进和新功能增加。TiDB 默认收集使用情况信息，并分享给 PingCAP 用于改善产品。新增功能包括支持 INSERT 语句中使用 MEMORY_QUOTA() hint，基于 TLS 证书 SAN 属性的登录认证，以及其他函数和表的新增配置项。Bug 修复包括执行计划不正确、运行时错误、内存统计过大等问题。PD、TiKV、TiFlash 和 TiCDC 也有相关改进和修复。 Tools 中 Backup & Restore (BR) 提升了多表场景下的恢复数据性能。
```
Comment (lilin90) [inline releases/release-4.0.2.md:4]:
```suggestion
summary: TiDB 4.0.2 发布，兼容性改进和新功能增加。TiDB 默认收集使用情况信息，并分享给 PingCAP 用于改善产品。新增功能包括支持 INSERT 语句中使用 MEMORY_QUOTA() hint，基于 TLS 证书 SAN 属性的登录认证，以及其他函数和表的新增配置项。Bug 修复包括执行计划不正确、运行时错误、内存统计过大等问题。PD、TiKV、TiFlash 和 TiCDC 也有相关改进和修复。Tools 中 Backup & Restore (BR) 提升了多表场景下的恢复数据性能。
```

File: releases/release-4.0.8.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB 4.0.8 Release Notes
+summary: TiDB 4.0.8 发布，新增聚合函数 `APPROX_PERCENTILE` 和 `CAST` 函数下推支持。优化了索引组合计算表达式选择率的贪心算法，记录更多的 RPC 信息，提升慢查询性能。修复了多个 BUG，如分区表可能遇到非预期 Panic、外连接时 Index Merge Join 结果不正确等。 PD 修复了 TiDB Dashboard 引起 PD panic 的错误。 TiFlash 修复了多个问题，如日志信息中时间戳错误、重启后可能提示数据文件损坏等。 Backup & Restore (BR) 修复了 Restore 期间可能发生的 `send on closed channel` panic 问题。
```
Comment (lilin90) [inline releases/release-4.0.8.md:3]:
```suggestion
summary: TiDB 4.0.8 发布，新增聚合函数 `APPROX_PERCENTILE` 和 `CAST` 函数下推支持。优化了索引组合计算表达式选择率的贪心算法，记录更多的 RPC 信息，提升慢查询性能。修复了多个 BUG，如分区表可能遇到非预期 Panic、外连接时 Index Merge Join 结果不正确等。 PD 修复了 TiDB Dashboard 引起 PD panic 的错误。TiFlash 修复了多个问题，如日志信息中时间戳错误、重启后可能提示数据文件损坏等。Backup & Restore (BR) 修复了 Restore 期间可能发生的 `send on closed channel` panic 问题。
```

File: releases/release-5.1.2.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB 5.1.2 Release Notes
+summary: TiDB 5.1.2 发布，包含兼容性更改、改进提升、Bug 修复等内容。兼容性更改包括修复多个 Bug，改进提升包括根据直方图行数触发 auto-analyze、支持动态更改 TiCDC 配置项等。Bug 修复涉及 hash 列为 ENUM 类型时 index hash join 的结果可能出错、TiKV 从 v3.x 升级至较高版本后出现 Panic 等问题。 Tools 方面的改进包括 BR 修复备份数据和恢复数据时显示的平均速度数值不准确的问题、Dumpling 修复特定 MySQL 版本下导致 dump 阶段卡死的问题等。
```
Comment (lilin90) [inline releases/release-5.1.2.md:3]:
```suggestion
summary: TiDB 5.1.2 发布，包含兼容性更改、改进提升、Bug 修复等内容。兼容性更改包括修复多个 Bug，改进提升包括根据直方图行数触发 auto-analyze、支持动态更改 TiCDC 配置项等。Bug 修复涉及 hash 列为 ENUM 类型时 index hash join 的结果可能出错、TiKV 从 v3.x 升级至较高版本后出现 Panic 等问题。Tools 方面的改进包括 BR 修复备份数据和恢复数据时显示的平均速度数值不准确的问题、Dumpling 修复特定 MySQL 版本下导致 dump 阶段卡死的问题等。
```

File: releases/release-rc.4.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB RC4 Release Notes
 aliases: ['/docs-cn/dev/releases/release-rc.4/','/docs-cn/dev/releases/rc4/']
+summary: TiDB RC4 版本对 MySQL 兼容性、SQL 优化器、系统稳定性、性能做了大量工作。重点优化了写入速度，计算任务调度支持优先级，避免分析型大事务影响在线事务。SQL 优化器全新改版，查询代价估算更准确，且能自动选择 Join 物理算子。功能方面进一步 MySQL 兼容性。同时开源了 TiSpark 项目，可以通过 Spark 读取和分析 TiKV 中的数据。 PD 支持通过 PD 设置 TiKV location labels，调度优化，优化数据加载，加快 failover 速度。 TiKV 支持查询优先级设置，支持 RC 隔离级别，完善 Jepsen，支持 Document Store，提升性能，提升稳定性。 TiSpark Beta Release 支持谓词下推，支持聚合下推，支持范围裁剪。
```
Comment (lilin90) [inline releases/release-rc.4.md:4]:
```suggestion
summary: TiDB RC4 版本对 MySQL 兼容性、SQL 优化器、系统稳定性、性能做了大量工作。重点优化了写入速度，计算任务调度支持优先级，避免分析型大事务影响在线事务。SQL 优化器全新改版，查询代价估算更准确，且能自动选择 Join 物理算子。功能方面进一步 MySQL 兼容性。同时开源了 TiSpark 项目，可以通过 Spark 读取和分析 TiKV 中的数据。PD 支持通过 PD 设置 TiKV location labels，调度优化，优化数据加载，加快 failover 速度。 TiKV 支持查询优先级设置，支持 RC 隔离级别，完善 Jepsen，支持 Document Store，提升性能，提升稳定性。TiSpark Beta Release 支持谓词下推，支持聚合下推，支持范围裁剪。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**